src/views/research/research.jsx

```diff
@@ -11,20 +11,26 @@ const Page = require('../../components/page/www/page.jsx');
 const Research = props => (
     <InformationPage title={props.intl.formatMessage({id: 'research.title'})}>
         <div className="inner info-inner">
-            <p><FormattedMessage id="research.conductors" /></p>
+            <p><FormattedMessage
+                id="research.conductors"
+                values={{
+                    ethicsCodeLink: (
+                        <a href="ETHICS CODE LINK">
```

Comment:
Once it exists, this will be the route.
```suggestion
                        <a href="/code_of_ethics/">
```